src/app/notifications/notifications.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngFor="let notification of allNotifications$ | async">
+  <div class="container text-center notification-bar {{ notification.priority }}">
```

Comment:
Since we're using the notifications endpoint for both sitewide banner stuff and news/updates pieces, you should filter the notifications by type.  After you loop through with ngFor, you can add an ngIf to the next div to filter out by type.

Reply:
Alternatively, you could have `bannerNotifications$` in notifications.query.ts and then you don't need the filtering logic in the HTML.

Reply:
Added a *ngIf to check if the notification is SITEWIDE. For now, if the notification is not SITEWIDE, then it does nothing.